rolling/distribution.yaml

```
@@ -8621,6 +8621,12 @@ repositories:
         release: release/rolling/{package}/{version}
       url: https://github.com/ros2-gbp/topic_based_ros2_control-release.git
       version: 0.2.0-2
+  topic_based_hardware_interfaces:
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros2-gbp/topic_based_hardware_interfaces-release.git
+      version: 0.1.0-1
```

```suggestion
  topic_based_hardware_interfaces:
    doc:
      type: git
      url: https://github.com/ros-controls/topic_based_hardware_interfaces.git
      version: main
    source:
      type: git
      url: https://github.com/ros-controls/topic_based_hardware_interfaces.git
      version: main
    status: developed
```
copy pasted the above